src/components/view/ReservationForm.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+import { createReservation } from '../../redux/ReservationsSlice';
+import { fetchLocations } from '../../redux/LocationsSlice';
+import { fetchServices } from '../../redux/ServicesSlice';
+
+const ReservationForm = () => {
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const locations = useSelector((state) => state.locations.locations);
+  const services = useSelector((state) => state.services.services);
```

Comment:
- [ ] Kindly use `const services = useSelector((state) => state.movies.movies)` since we changed the name of our `MovieSlice initialState`  to movies